doc/newsfragments/2626_changed.pattern_with_parts.rst

```diff
@@ -0,0 +1,5 @@
+Filtering Patterns now are easier to use with MultiTest with parts.
+
+    * Both patterns with part specified or not could be used to filter MultiTest with parts.
+    * Applying patterns won't cause certain testcase appear in some different part now.
```

Comment:
Applying filtering pattern will not change parts shuffling - the same testcase always end up in the same part of a MultiTest.